pkg/linters/license/license.go

```diff
@@ -43,7 +42,8 @@ func (o *Copyright) Run(m *module.Module) (errors.LintRuleErrorsList, error) {
 	for _, fileName := range files {
 		name, _ := strings.CutPrefix(fileName, m.GetPath())
 		name = m.GetName() + ":" + name
-		if slices.Contains(o.cfg.CopyrightExcludes, name) {
+
+		if fsutils.FileNameMatchAnyMask(name, o.cfg.CopyrightExcludes) {
```

Comment:
```
name = m.GetName() + ":" + name
```

This means that the name is `moduleName:/hooks/.venv/**/*`
But in tests, we only test the pattern path `/hooks/.venv/filename.txt`

This means that exceptions in the form that are being set now will not work.

Reply:
Think of `FileNameMatchAnyMask` function as masks not for file paths, but just strings. I will rename these functions to clear up any misunderstandings. :-)